src/main/java/com/shopping_app/shoppingApp/model/Request/ProductRequest.java

```diff
@@ -0,0 +1,54 @@
+package com.shopping_app.shoppingApp.model.Request;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.*;
+
+import javax.validation.constraints.NotBlank;
+
+
+
```

Comment:
Please make sure not to leave empty lines where it's not required, in other files as well